### ISSUE TITLE
✨ Include custom sizing in radial menu

### ISSUE THREAD

**Issue by JaceThings** [MrKai77/Loop#157]
### Feature Request Description

With the addition of custom sizes, I think it would be cool if the radial menu displayed them, even if they are not part of the default window sizes.

For example: 
Any position listed under "Quarters", the radial menu will have a selected corner when the window is in that position.

![002442@2x](https://github.com/MrKai77/Loop/assets/60551230/ceabf087-d437-4e4f-ad9f-74a24151e957)

When a custom size, such as my own custom "Sixths," is selected, no indicator is provided. However, it is possible to calculate the filled area by using the given percentage or by calculating the percentage based on the given `px` value divided by the screen width or height.

![002443](https://github.com/MrKai77/Loop/assets/60551230/cd72d1bd-d97c-43cb-9657-c7f7d209b12a)

### Screenshots

https://github.com/MrKai77/Loop/assets/60551230/60d41783-fe9e-4d17-8d5f-8ae2460d0b09

### Final Checks

- [X] My issue title is descriptive
- [X] This is a single feature request (multiple feature requests should be reported individually)

**Comment by MrKai77** [comment]
Good idea! I think an even easier way to do this would be to use the provided anchor as the direction shown in the radial menu!

**Comment by JaceThings** [comment]
The anchor would help, I was more interested in the coloured areas being a 1:1 % match 😅

66% of your screen, on the right, would fill 66% of the right side of the menu, etc

**Comment by Anatharias** [comment]
I second this. on a 34 or 38" ultawide display, sixth are fantastic
The sixth should be added to the radial menu, with a checkbox to enable them, for instance.
The same goes for the custom sizes.

Actually, there should be a screen where you can customize the radial options with whatever prefered size you created.

For instance, I want custom sizes in sixth, but some sixth areas too large on a 38"
so Teams and Whatsapp would have their own right-hand area (28% x 50%), 28% for teams is the minimum the window must be to display recent contacts + convo at once and 33% is clearly not useful for Teams, same goes for whatsapp.

The remainder of the screen 72%, would be available in 36% vertical chunks or 36% x 50%, so I would have a larger area than a sixth for the two other sections.

Add this, and I can ditch the Rectangle app, lol.
I could do it already, actually, but for a broader adoption, having it available in the Radial menu would make is easier to handle. Some people just can't deal with shortcuts

Thanks !

**Comment by tofagerl** [comment]
I third(!) this. 
The only challenge is how these sizes should be displayed, and imo it should be possible to simply overwrite the existing areas of the circle. 
For example: I have an ultrawide screen, and it's logical to me, though probably not to anyone else, that this is the only logical arrangement: 
Top: Maximized (not full screen, I hate the animation delays)
Bottom: 1/3 size in the middle.
Upper right: 2/3 size anchored to the right.
Middle right: 1/2 size anchored to the right.
Lower right: 1/3 size anchored to the right.
Upper left: 2/3 size anchored to the left.
Middle left: 1/2 size anchored to the left.
Lower left: 1/3 size anchored to the left.

This is clearly just my particular logic playing in, but if we could figure out a way to make this configurable, that'd be great :)

**Comment by JaceThings** [comment]
God I'd love to see an image of that setup, it seems so oddly specific 😭

**Comment by tofagerl** [comment]
It really is one of those "how did I end up like this?" habits, yes... ;p

**Comment by marcosvrs** [comment]
The only reason why I haven't decided to make Loop my main window manager in MacOS is this missing feature. Looking forward!

**Comment by MrKai77** [comment]
It seems quite a few people have been following this thread to see if radial menu customization (not custom sizing in the UI) would be added. I’m happy to say that radial menu customization for actions is now available under the “Advanced” tab in Loop!

As for this issue, it is still on the roadmap, but it’s mainly a visual enhancement rather than something that affects functionality or usability, so it isn’t currently considered high-priority :)
